server/conf/application.conf

```diff
@@ -229,6 +229,9 @@ play.http {
   # If null, will attempt to load a class called ErrorHandler in the root package,
   errorHandler = "controllers.ErrorHandler"
 
+  ## Max memory
+  parser.maxMemoryBuffer = 256K
```

Comment:
make a custom body parser instead of setting this globally: https://www.playframework.com/documentation/2.9.x/JavaBodyParsers#Writing-a-custom-max-length-body-parser